daemon/protocol/daemon.proto

```diff
@@ -19,6 +21,7 @@ service ScootDaemon {
   rpc CheckoutSnapshot(CheckoutSnapshotRequest) returns (CheckoutSnapshotReply) {}
   rpc Run(RunRequest) returns (RunReply) {}
   rpc Poll(PollRequest) returns (PollReply) {}
+  rpc StopDaemon(EmptyStruct) returns (EmptyStruct) {}
```

Comment:
I tried without parameters and return, protobuf does not accept that. 
I prefer EmptyStruct since it can be reused if we add any other commands that don't need parameters.